server/lib/src/entry.rs

```diff
@@ -53,20 +54,17 @@ use kanidm_proto::v1::Entry as ProtoEntry;
 use ldap3_proto::simple::{LdapPartialAttribute, LdapSearchResultEntry};
 use openssl::ec::EcKey;
 use openssl::pkey::{Private, Public};
+use std::cmp::Ordering;
+pub use std::collections::BTreeSet as Set;
```

Comment:
I really hate the idea of having to re-publish a std thing at all, but why rename it? 🤮 Similarly, the rename of `BTreeMap` to `Map` below - it adds to cognitive overhead while looking at this code - especially since it's being imported twice?

Reply:
This was in the past when I wrote the code to allow swapping between BTreeSet and HashSet for testing. It's something we can undo in a separate PR.

Reply:
k, u better